src/withDirection.jsx

```diff
@@ -23,16 +23,19 @@ const defaultDirection = DIRECTIONS.LTR;
 
 // export for convenience, in order for components to spread these onto their propTypes
 export const withDirectionPropTypes = {
-  direction: directionPropType.isRequired,
+  direction: directionPropType,
```

Comment:
we went out of our way to leave this required when considering AutoDirectionProvider - the likelihood of a bug with someone forgetting to provide it is *very* high.

Anything wrapped in `withDirection` *must* get a direction, that's the sole purpose of the HOC.